App/Screens/About/Language/Language.tsx

```diff
@@ -49,15 +49,15 @@ export function Language(): React.ReactElement {
 	return (
 		<View style={styles.container}>
 			<Text style={theme.link}>
-				{names[i18n.locale as keyof typeof names].nativeName}
+				{names[i18n.language as keyof typeof names].nativeName}
```

Comment:
This `<Language/>` component is actually hidden right now, but thanks for taking care of this file. I will also play a bit on my side.

Reply:
So, should I touch this part?

Reply:
Yes, keep it like this, i will test it after this PR is merged.